lms/templates/admin/organization.html.jinja2

```diff
@@ -0,0 +1,34 @@
+{% import "macros.jinja" as macros %}
+
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% set public_id = org.public_id(request.region) %}
+{% block header %}
+    Organization {{org.name}} / {{ public_id }}
+{% endblock %}
+
+{% block content %}
+    <fieldset class="box">
+    <legend class="label has-text-centered">Organization</legend>
+    <form method="POST" action="{{ request.route_url("admin.organization", public_id=public_id) }}">
+        <input type="hidden" name="csrf_token" value="{{get_csrf_token()}}">
+        {{ macros.disabled_text_field("ID", public_id) }}
+        {{ macros.form_text_field(request, "Name", "name", org.name) }}
+        {% call macros.field_body("Enabled") %}
+            <label class="checkbox">
+                <input {% if org.enabled %}checked{% endif%} type="checkbox"
+                    name="enabled" disabled>
+            </label>
+        {% endcall %}
+        {{ macros.created_updated_fields(org) }}
+        <div class="has-text-right mb-6">
+            <input type="submit" class="button is-primary" value="Save" />
+        </div>
+    </form>
+
+    <legend class="label has-text-centered">Organization application instances</legend>
+        {% if org.application_instances %}
+            {{ macros.instances_table(request, org.application_instances) }}
```

Comment:
It would be very good to include some human readable fields for the org, like:

 * tool_consumer_instance_name
 * tool_consumer_instance_contact_email
 * lms_url
 
As these are the fields that would let you know as a human which is which. Currently it's all opaque ids.

I think this is particularly important in the case of orgs, as we are looking for ones which don't belong, and without clicking through to each, there's not much to go on.

If there's not enough room, I think we could afford to drop a few of the opaque fields to make room.

Reply:
Added name and url, removed GUID.